### ISSUE TITLE
翻译统一用词，欢迎在底下补充

### ISSUE THREAD

**Issue by MARD1NO** [Oneflow-Inc/oneflow-documentation#162]
显卡->device
数据并行 -> data parallelism
调度节点 -> centralized scheduler
机制 -> mechanism
代码 -> code
代码解析 -> code explanation
计算图 -> computation graph
功能特性 -> features
数据搬运 -> data movement
反序列化 -> deserialize
线性加速比 -> linear speedup
编译阶段 -> compile time
混合并行 -> hybrid parallelism
数据流水线 -> data pipeline
数据准备 -> data prepare
文件名补齐 -> complete
精调 -> fine-tuning
迁移学习 ->transfer learning

**Comment by xiejiachen** [comment]
模型构图 -> model graph
> https://www.tensorflow.org/api_docs/python/tf/Graph
调度节点 -> scheduler/或 scheduler node
